[BUG]启动不起来，已知解决方案，改默认端口号至1000以下的系统端口字段

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

NextGen

### Operating System

Windows 10

### Minecraft Version

1.21.4

### Describe the bug

我期望正常启动但是报错了

[错误报告-11-9-2025_下午_9.12.19.zip](https://github.com/user-attachments/files/22275981/-11-9-2025_._9.12.19.zip)

### Steps to reproduce

启动游戏

### Client Log

```shell
[21:11:41] [main/INFO]: Loading Minecraft 1.21.4 with Fabric Loader 0.17.2
[21:11:41] [ForkJoinPool-1-worker-9/WARN]: Mod com_github_oryxel_cubeconverter uses the version 916cadb12a which isn't compatible with Loader's extended semantic version format (Could not parse version number component '916cadb12a'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[21:11:41] [ForkJoinPool-1-worker-1/WARN]: Mod org_cloudburstmc_netty_netty-transport-raknet uses the version 1.0.0.CR3-SNAPSHOT which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'CR3'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[21:11:41] [main/INFO]: Loading 140 mods:
	- fabric-api 0.119.4+1.21.4
	   |-- fabric-api-base 0.4.54+b47eab6b04
	   |-- fabric-api-lookup-api-v1 1.6.86+b1caf1e904
	   |-- fabric-biome-api-v1 15.0.6+b1c29d8e04
	   |-- fabric-block-api-v1 1.0.31+7feeb73304
	   |-- fabric-block-view-api-v2 1.0.20+9c49cc8c04
	   |-- fabric-blockrenderlayer-v1 2.0.8+7feeb73304
	   |-- fabric-client-tags-api-v1 1.1.29+20ea1e2304
	   |-- fabric-command-api-v1 1.2.62+f71b366f04
	   |-- fabric-command-api-v2 2.2.41+e496eb1504
	   |-- fabric-commands-v0 0.2.79+df3654b304
	   |-- fabric-content-registries-v0 9.1.19+25d1a67604
	   |-- fabric-convention-tags-v1 2.1.20+7f945d5b04
	   |-- fabric-convention-tags-v2 2.14.1+aebda09404
	   |-- fabric-crash-report-info-v1 0.3.6+7feeb73304
	   |-- fabric-data-attachment-api-v1 1.6.2+e99da0f704
	   |-- fabric-data-generation-api-v1 22.3.1+0f4e5f5504
	   |-- fabric-dimensions-v1 4.0.10+7feeb73304
	   |-- fabric-entity-events-v1 2.0.15+62245bef04
	   |-- fabric-events-interaction-v0 4.0.4+a4eebcf004
	   |-- fabric-game-rule-api-v1 1.0.63+7d48d43904
	   |-- fabric-item-api-v1 11.4.0+189dd6fe04
	   |-- fabric-item-group-api-v1 4.2.2+fcb9601404
	   |-- fabric-key-binding-api-v1 1.0.57+7d48d43904
	   |-- fabric-keybindings-v0 0.2.55+df3654b304
	   |-- fabric-lifecycle-events-v1 2.5.4+bf2a60eb04
	   |-- fabric-loot-api-v2 3.0.38+3f89f5a504
	   |-- fabric-loot-api-v3 1.0.26+203e6b2304
	   |-- fabric-message-api-v1 6.0.26+238a33c004
	   |-- fabric-model-loading-api-v1 4.3.0+ae23723504
	   |-- fabric-networking-api-v1 4.4.0+db5e668204
	   |-- fabric-object-builder-api-v1 18.0.14+38b0d59804
	   |-- fabric-particles-v1 4.0.14+7feeb73304
	   |-- fabric-recipe-api-v1 8.1.1+640e77ae04
	   |-- fabric-registry-sync-v0 6.1.11+4a9c1ece04
	   |-- fabric-renderer-api-v1 5.0.3+50f0feb204
	   |-- fabric-renderer-indigo 2.0.3+50f0feb204
	   |-- fabric-rendering-data-attachment-v1 0.3.58+73761d2e04
	   |-- fabric-rendering-fluids-v1 3.1.19+7feeb73304
	   |-- fabric-rendering-v1 10.2.1+0d31b09f04
	   |-- fabric-resource-conditions-api-v1 5.0.13+203e6b2304
	   |-- fabric-resource-loader-v0 3.1.1+360374ac04
	   |-- fabric-screen-api-v1 2.0.38+7feeb73304
	   |-- fabric-screen-handler-api-v1 1.3.118+7feeb73304
	   |-- fabric-sound-api-v1 1.0.32+7feeb73304
	   |-- fabric-tag-api-v1 1.0.7+7d48d43904
	   |-- fabric-transfer-api-v1 5.4.9+efa825c904
	   \-- fabric-transitive-access-wideners-v1 6.3.2+56e78b9b04
	- fabric-language-kotlin 1.13.5+kotlin.2.2.10
	   |-- org_jetbrains_kotlin_kotlin-reflect 2.2.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib 2.2.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 2.2.10
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 2.2.10
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.29.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.10.2
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.10.2
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.7.1
	   |-- org_jetbrains_kotlinx_kotlinx-io-bytestring-jvm 0.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-io-core-jvm 0.8.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.9.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.9.0
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.9.0
	- fabricloader 0.17.2
	   \-- mixinextras 0.5.0
	- iris 1.8.8+mc1.21.4
	   |-- io_github_douira_glsl-transformer 2.0.1
	   |-- org_anarres_jcpp 1.4.14
	   \-- org_antlr_antlr4-runtime 4.13.1
	- java 21
	- liquidbounce 1.9.2
	   |-- ai_djl_api 0.34.0
	   |-- ai_djl_pytorch_pytorch-engine 0.34.0
	   |-- com_github_ccbluex_discordipc 4.0.0
	   |-- com_github_ccbluex_mc-authlib 1.4.1
	   |-- com_kohlschutter_junixsocket_junixsocket-common 2.8.3
	   |-- com_kohlschutter_junixsocket_junixsocket-native-common 2.8.3
	   |-- com_squareup_okhttp3_okhttp-coroutines 5.1.0
	   |-- com_squareup_okhttp3_okhttp-jvm 5.1.0
	   |-- com_squareup_okio_okio-jvm 3.15.0
	   |-- com_thealtening_api_api 4.1.0
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- io_netty_netty-codec-socks 4.1.115
	   |-- io_netty_netty-handler-proxy 4.1.115
	   |-- io_netty_netty-transport-classes-kqueue 4.1.115
	   |-- io_netty_netty-transport-native-kqueue_osx-aarch_64 4.1.115
	   |-- io_netty_netty-transport-native-kqueue_osx-x86_64 4.1.115
	   |-- mcef 3.1.2-1.21.4
	   |-- net_ccbluex_netty-httpserver 2.3.2
	   |-- net_fabricmc_tiny-mappings-parser 0.3.0+build.17
	   |-- org_ahocorasick_ahocorasick 0.6.3
	   |-- org_apache_tika_tika-core 2.9.2
	   |-- org_graalvm_js_js-language 24.2.1
	   |-- org_graalvm_polyglot_polyglot 24.2.1
	   |-- org_graalvm_regex_regex 24.2.1
	   |-- org_graalvm_sdk_collections 24.2.1
	   |-- org_graalvm_sdk_jniutils 24.2.1
	   |-- org_graalvm_sdk_nativeimage 24.2.1
	   |-- org_graalvm_sdk_word 24.2.1
	   |-- org_graalvm_shadowed_icu4j 24.2.1
	   |-- org_graalvm_shadowed_json 24.2.1
	   |-- org_graalvm_shadowed_xz 24.2.1
	   |-- org_graalvm_tools_chromeinspector-tool 24.2.1
	   |-- org_graalvm_tools_coverage-tool 24.2.1
	   |-- org_graalvm_tools_dap-tool 24.2.1
	   |-- org_graalvm_tools_insight-heap-tool 24.2.1
	   |-- org_graalvm_tools_insight-tool 24.2.1
	   |-- org_graalvm_tools_lsp-tool 24.2.1
	   |-- org_graalvm_tools_lsp_api 24.2.1
	   |-- org_graalvm_tools_profiler-tool 24.2.1
	   |-- org_graalvm_truffle_truffle-api 24.2.1
	   |-- org_graalvm_truffle_truffle-compiler 24.2.1
	   \-- org_graalvm_truffle_truffle-runtime 24.2.1
	- minecraft 1.21.4
	- sodium 0.6.13+mc1.21.4
	- viafabricplus 4.0.5-BACKPORT
	   |-- com_github_oryxel_cubeconverter 916cadb12a
	   |-- com_google_code_findbugs_jsr305 3.0.2
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- com_viaversion_viaaprilfools-common 4.0.2-20250426.140810-22
	   |-- com_viaversion_viabackwards-common 5.3.3-20250426.185209-1
	   |-- com_viaversion_vialoader 4.0.3-20250426.141011-6
	   |-- com_viaversion_viaversion-common 5.3.3-20250426.143637-8
	   |-- de_florianmichael_classic4j 2.1.1-SNAPSHOT
	   |-- io_jsonwebtoken_jjwt-api 0.12.6
	   |-- io_jsonwebtoken_jjwt-gson 0.12.6
	   |-- io_jsonwebtoken_jjwt-impl 0.12.6
	   |-- io_netty_netty-codec-http 4.2.0
	   |-- net_jodah_expiringmap 0.5.10
	   |-- net_lenni0451_commons_httpclient 1.6.0
	   |-- net_lenni0451_mcstructs-bedrock_forms 2.0.0
	   |-- net_lenni0451_mcstructs-bedrock_text 2.0.0
	   |-- net_lenni0451_reflect 1.4.0
	   |-- net_raphimc_minecraftauth 4.1.1
	   |-- net_raphimc_viabedrock 0.0.16-20250404.165607-5
	   |-- net_raphimc_vialegacy 3.0.10-20250426.140714-8
	   |-- org_cloudburstmc_netty_netty-transport-raknet 1.0.0.CR3-SNAPSHOT
	   |-- org_lz4_lz4-pure-java 1.8.0
	   |-- team_unnamed_mocha 3.0.1
	   |-- viafabricplus-api 4.0.5-BACKPORT
	   |-- viafabricplus-api-legacy 4.0.5-BACKPORT
	   \-- viafabricplus-visuals 4.0.5-BACKPORT
[21:11:42] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.7 Source=file:/C:/Users/GardeniaAdmin/Desktop/KG/JMclient/.minecraft/libraries/net/fabricmc/sponge-mixin/0.16.3+mixin.0.8.7/sponge-mixin-0.16.3+mixin.0.8.7.jar Service=Knot/Fabric Env=CLIENT
[21:11:42] [main/INFO]: Compatibility level set to JAVA_21
[21:11:42] [main/WARN]: Reference map 'iris.refmap.json' for mixins.iris.json could not be read. If this is a development environment you can ignore this message
[21:11:42] [main/WARN]: Reference map 'iris-fabric.refmap.json' for mixins.iris.fabric.json could not be read. If this is a development environment you can ignore this message
[21:11:42] [main/WARN]: Reference map 'iris.refmap.json' for iris-batched-entity-rendering.mixins.json could not be read. If this is a development environment you can ignore this message
[21:11:42] [main/WARN]: Reference map 'iris.refmap.json' for mixins.iris.fantastic.json could not be read. If this is a development environment you can ignore this message
[21:11:42] [main/WARN]: Reference map 'iris.refmap.json' for mixins.iris.vertexformat.json could not be read. If this is a development environment you can ignore this message
[21:11:42] [main/WARN]: Reference map 'iris.refmap.json' for mixins.iris.bettermipmaps.json could not be read. If this is a development environment you can ignore this message
[21:11:42] [main/WARN]: Reference map 'iris.refmap.json' for mixins.iris.compat.sodium.json could not be read. If this is a development environment you can ignore this message
[21:11:42] [main/WARN]: Reference map 'iris.refmap.json' for mixins.iris.fixes.maxfpscrash.json could not be read. If this is a development environment you can ignore this message
[21:11:42] [main/INFO]: Loaded configuration file for Sodium: 43 options available, 1 override(s) found
[21:11:42] [main/WARN]: Force-disabling mixin 'features.render.world.sky.FogRendererMixin' as rule 'mixin.features.render.world.sky' (added by mods [iris]) disables it and children
[21:11:42] [main/WARN]: Force-disabling mixin 'features.render.world.sky.ClientLevelMixin' as rule 'mixin.features.render.world.sky' (added by mods [iris]) disables it and children
[21:11:42] [main/WARN]: Force-disabling mixin 'features.render.world.sky.LevelRendererMixin' as rule 'mixin.features.render.world.sky' (added by mods [iris]) disables it and children
[21:11:42] [main/WARN]: Error loading class: net/caffeinemc/mods/lithium/common/entity/movement/ChunkAwareBlockCollisionSweeper (java.lang.ClassNotFoundException: net/caffeinemc/mods/lithium/common/entity/movement/ChunkAwareBlockCollisionSweeper)
[21:11:43] [main/INFO]: Searching for graphics cards...
[21:11:43] [main/INFO]: Found graphics adapter: AdapterInfo{vendor=NVIDIA, description='NVIDIA GeForce RTX 3060', adapterType=0x0000030B, openglIcdFilePath='C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_7621da800da0efae\nvoglv64.dll', openglIcdVersion=32.0.15.8108}
[21:11:43] [main/WARN]: Sodium has applied one or more workarounds to prevent crashes or other issues on your system: [NVIDIA_THREADED_OPTIMIZATIONS_BROKEN]
[21:11:43] [main/WARN]: This is not necessarily an issue, but it may result in certain features or optimizations being disabled. You can sometimes fix these issues by upgrading your graphics driver.
[21:11:43] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.5.0).
[21:11:44] [Datafixer Bootstrap/INFO]: 243 Datafixer optimizations took 444 milliseconds
[21:11:49] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[21:11:49] [Render thread/INFO]: Setting user: Gardenia
[21:11:49] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[21:11:50] [Render thread/INFO]: Item Groups: [ Heads, Exploits, Containers ]
[21:11:50] [Via-Mappingloader-0/INFO]: Loading block connection mappings ...
[21:11:50] [Via-Mappingloader-0/INFO]: Using FastUtil Long2ObjectOpenHashMap for block connections
[21:11:50] [ForkJoinPool.commonPool-worker-1/INFO]: Loading translations...
[21:11:50] [ForkJoinPool.commonPool-worker-1/INFO]: Registering protocols...
[21:11:50] [Download-2/ERROR]: Failed to fetch user properties
com.mojang.authlib.exceptions.InvalidCredentialsException: Status: 401
	at knot/com.mojang.authlib.exceptions.MinecraftClientHttpException.toAuthenticationException(MinecraftClientHttpException.java:59) ~[authlib-6.0.57.jar:?]
	at knot/com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:150) ~[authlib-6.0.57.jar:?]
	at knot/net.minecraft.class_310.method_55608(class_310.java:457) ~[client-intermediary.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at knot/com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:103) ~[authlib-6.0.57.jar:?]
	at knot/com.mojang.authlib.minecraft.client.MinecraftClient.get(MinecraftClient.java:56) ~[authlib-6.0.57.jar:?]
	at knot/com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:121) ~[authlib-6.0.57.jar:?]
	... 5 more
[21:11:50] [Render thread/INFO]: Backend library: LWJGL version 3.3.3-snapshot
[21:11:50] [Render thread/INFO]: Modifying process environment to apply workarounds for the NVIDIA graphics driver...
[21:11:51] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[21:11:51] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce RTX 3060/PCIe/SSE2
[21:11:51] [Render thread/INFO]: OpenGL Version: 3.2.0 NVIDIA 581.08
[21:11:51] [Render thread/INFO]: Modifying OpenGL context to apply workarounds for the NVIDIA graphics driver...
[21:11:51] [ForkJoinPool.commonPool-worker-1/INFO]: Started resource pack HTTP server on http://127.0.0.1:1113/
[21:11:51] [Render thread/INFO]: Enabling GL_DEBUG_OUTPUT_SYNCHRONOUS to force the NVIDIA driver to disable threadedcommand submission
[21:11:51] [Render thread/WARN]: RivaTuner Statistics Server (RTSS) has injected into the process! Attempting to apply workarounds for compatibility...
[21:11:51] [Render thread/INFO]: Searching directory: C:\Program Files (x86)\RivaTuner Statistics Server
[21:11:51] [Render thread/INFO]: Parsing file: C:\Program Files (x86)\RivaTuner Statistics Server\RTSS.exe
[21:11:51] [Render thread/INFO]: Detected RivaTuner Statistics Server version: 7.3.5.28211
[21:11:51] [ForkJoinPool.commonPool-worker-1/INFO]: ViaVersion detected lowest supported version by the proxy: c0.0.15a-1 (0)
[21:11:51] [ForkJoinPool.commonPool-worker-1/INFO]: Highest supported version by the proxy: 1.21.5 (770)
[21:11:51] [ForkJoinPool.commonPool-worker-1/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[21:11:51] [Render thread/INFO]: Debug functionality is disabled.
[21:11:51] [Render thread/INFO]: ARB_direct_state_access detected, enabling DSA.
[21:11:51] [Render thread/INFO]: Shaders are disabled because enableShaders is set to false in iris.properties
[21:11:51] [Render thread/WARN]: Removed resource pack file/! §5Lavender§8[§716x§8]1.21.7.zip from options because it doesn't seem to exist anymore
[21:11:51] [Render thread/WARN]: Removed resource pack file/白色附魔.zip from options because it doesn't seem to exist anymore
[21:11:51] [Render thread/WARN]: Removed resource pack file/打击音效玻璃1.21.7.zip from options because it doesn't seem to exist anymore
[21:11:52] [Render thread/INFO]: Hardware information:
[21:11:52] [Render thread/INFO]: CPU: 12x AMD Ryzen 5 5600 6-Core Processor 
[21:11:52] [Render thread/INFO]: GPU: NVIDIA GeForce RTX 3060/PCIe/SSE2 (Supports OpenGL 3.2.0 NVIDIA 581.08)
[21:11:52] [Render thread/INFO]: OS: Windows 10 (10.0)
[21:11:52] [Via Async Scheduler 0/INFO]: Finished mapping loading, shutting down loader executor.
[21:11:52] [Render thread/INFO]: Launching JMcomicFix v1.9.2 by CCBlueX
[21:11:52] [Render thread/INFO]: Client Version: 1.9.2 (git-63d021c)
[21:11:52] [Render thread/INFO]: Client Branch: nextgen
[21:11:52] [Render thread/INFO]: Operating System: Windows 10 (10.0)
[21:11:52] [Render thread/INFO]: Java Version: 21.0.7
[21:11:52] [Render thread/INFO]: Screen Resolution: 854x480
[21:11:52] [Render thread/INFO]: Refresh Rate: 200 Hz
[21:11:52] [ForkJoinPool.commonPool-worker-2/WARN]: [Iris Update Check] This version doesn't have an update index, skipping.
[21:11:52] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric, com_viaversion_viaaprilfools-common, com_viaversion_viabackwards-common, com_viaversion_viaversion-common, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-tag-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, iris, net_raphimc_viabedrock, net_raphimc_vialegacy, sodium, viafabricplus, viafabricplus-visuals
[21:11:52] [Worker-Main-8/INFO]: Found unifont_jp_patch-16.0.01.hex, loading
[21:11:52] [Worker-Main-4/INFO]: Found unifont_all_no_pua-16.0.01.hex, loading
[21:11:54] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on 耳机 (K5 TX)
[21:11:54] [Render thread/INFO]: Sound engine started
[21:11:54] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[21:11:54] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[21:11:54] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[21:11:54] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[21:11:54] [Render thread/INFO]: Created: 2048x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[21:11:54] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[21:11:54] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[21:11:54] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[21:11:54] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[21:11:54] [Render thread/INFO]: Created: 64x64x0 minecraft:textures/atlas/map_decorations.png-atlas
[21:11:54] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[21:11:54] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/paintings.png-atlas
[21:11:54] [Render thread/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[21:11:54] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/gui.png-atlas
[21:11:54] [Render thread/INFO]: Sound initialized: duration=2786ms
[21:11:54] [Render thread/INFO]: Initial session saved: Gardenia (00000000-0000-3008-999a-572a2f95a539)
[21:11:54] [Render thread/INFO]: Probing environment...
[21:11:54] [Render thread/INFO]: Intermediary environment detected.
[21:11:54] [Render thread/INFO]: [ScriptAPI] Engine Version: Development Build, Supported languages: [ js ]
[21:11:55] [Render thread/INFO]: [STDOUT]: Failed to access title/subtitle fields: title
[21:11:55] [Render thread/INFO]: Loaded language en_us
[21:11:55] [Render thread/WARN]: VoidTP is missing fallback description key liquidbounce.module.voidTP.description
[21:11:55] [DefaultDispatcher-worker-3/INFO]: Initializing API...
[21:11:55] [DefaultDispatcher-worker-3/INFO]: API Session Token: q5I2azw9tMbU5E6J
[21:11:55] [DefaultDispatcher-worker-3/INFO]: Looking up available API endpoints...
[21:11:57] [DefaultDispatcher-worker-3/INFO]: API endpoint 'https://api.liquidbounce.net' is available
[21:11:57] [DefaultDispatcher-worker-5/INFO]: [Update] Update available: 1.9.2 -> 0.33.1
[21:11:58] [DefaultDispatcher-worker-7/INFO]: Successfully loaded Discord IPC configuration [443472046031110144].
[21:11:59] [DefaultDispatcher-worker-1/INFO]: Successfully loaded 956 heads from HeadDB
[21:11:59] [DefaultDispatcher-worker-1/INFO]: API initialization done.
[21:11:59] [Render thread/INFO]: Default port 10721 available.
[21:11:59] [Render thread/INFO]: Starting Netty server...
[21:11:59] [nioEventLoopGroup-3-1/INFO]: [id: 0xdf999d2d] REGISTERED
[21:11:59] [nioEventLoopGroup-3-1/INFO]: [id: 0xdf999d2d] BIND: 0.0.0.0/0.0.0.0:10721
[21:11:59] [nioEventLoopGroup-3-1/INFO]: [id: 0xdf999d2d] CLOSE
[21:11:59] [Render thread/ERROR]: Netty server failed - 10721
java.net.BindException: Address already in use: bind
	at java.base/sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at java.base/sun.nio.ch.Net.bind(Net.java:565) ~[?:?]
	at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(ServerSocketChannelImpl.java:344) ~[?:?]
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:301) ~[?:?]
	at knot/io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:141) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:561) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1281) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:600) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.handler.logging.LoggingHandler.bind(LoggingHandler.java:230) ~[netty-handler-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:602) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:922) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.channel.AbstractChannel.bind(AbstractChannel.java:259) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:380) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569) ~[netty-transport-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.115.Final.jar:?]
	at knot/io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.115.Final.jar:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
[21:11:59] [nioEventLoopGroup-3-1/INFO]: [id: 0xdf999d2d] UNREGISTERED
[21:11:59] [Render thread/INFO]: Shutting down Netty server...
[21:12:03] [Render thread/INFO]: Netty server stopped.
[21:12:03] [Render thread/ERROR]: Failed to bind to port 10721. Falling back to random port.
[21:12:03] [Render thread/INFO]: Starting Netty server...
[21:12:03] [nioEventLoopGroup-5-1/INFO]: [id: 0xfbf822fc] REGISTERED
[21:12:03] [nioEventLoopGroup-5-1/INFO]: [id: 0xfbf822fc] BIND: 0.0.0.0/0.0.0.0:16579
[21:12:03] [nioEventLoopGroup-5-1/INFO]: [id: 0xfbf822fc, L:/[0:0:0:0:0:0:0:0]:16579] ACTIVE
[21:12:03] [Render thread/INFO]: Netty server started on port 16579.
[21:12:03] [OkHttp http://127.0.0.1:10721/.../ERROR]: Failed to execute request GET http://127.0.0.1:10721/resource/liquidbounce/metadata.json)
java.net.ConnectException: Failed to connect to /127.0.0.1:10721
	at knot/okhttp3.internal.connection.ConnectPlan.connectSocket(ConnectPlan.kt:282) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.ConnectPlan.connectTcp(ConnectPlan.kt:140) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.FastFallbackExchangeFinder$launchTcpConnect$1.runOnce(FastFallbackExchangeFinder.kt:141) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.concurrent.TaskRunner$runnable$1.run(TaskRunner.kt:81) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.net.ConnectException: Connection refused: getsockopt
	at java.base/sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682) ~[?:?]
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592) ~[?:?]
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.base/java.net.Socket.connect(Socket.java:751) ~[?:?]
	at knot/okhttp3.internal.platform.Platform.connectSocket(Platform.kt:147) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.ConnectPlan.connectSocket(ConnectPlan.kt:280) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	... 6 more
[21:12:03] [Render thread/ERROR]: Failed to load theme metadata
java.net.ConnectException: Failed to connect to /127.0.0.1:10721
	at knot/okhttp3.internal.connection.ConnectPlan.connectSocket(ConnectPlan.kt:282) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.ConnectPlan.connectTcp(ConnectPlan.kt:140) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.FastFallbackExchangeFinder$launchTcpConnect$1.runOnce(FastFallbackExchangeFinder.kt:141) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.concurrent.TaskRunner$runnable$1.run(TaskRunner.kt:81) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.net.ConnectException: Connection refused: getsockopt
	at java.base/sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682) ~[?:?]
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592) ~[?:?]
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.base/java.net.Socket.connect(Socket.java:751) ~[?:?]
	at knot/okhttp3.internal.platform.Platform.connectSocket(Platform.kt:147) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.ConnectPlan.connectSocket(ConnectPlan.kt:280) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	... 6 more
[21:12:03] [Render thread/ERROR]: An error occurred!
java.lang.IllegalStateException: Failed to load theme metadata
	at knot/net.ccbluex.liquidbounce.integration.theme.Theme.loadMetadata(Theme.kt:67) ~[JMcomicFix_v1.9.2_rc50.jar:?]
	at knot/net.ccbluex.liquidbounce.integration.theme.Theme.access$loadMetadata(Theme.kt:47) ~[JMcomicFix_v1.9.2_rc50.jar:?]
	at knot/net.ccbluex.liquidbounce.integration.theme.Theme$loadMetadata$1.invokeSuspend(Theme.kt) ~[JMcomicFix_v1.9.2_rc50.jar:?]
	at knot/kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[org_jetbrains_kotlin_kotlin-stdlib-2.2.10-2020a97885177cae.jar:?]
	at knot/kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:98) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.2-dc3d13b93361afd1.jar:?]
	at knot/kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:263) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.2-dc3d13b93361afd1.jar:?]
	at knot/kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:94) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.2-dc3d13b93361afd1.jar:?]
	at knot/kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:70) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.2-dc3d13b93361afd1.jar:?]
	at knot/kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.2-dc3d13b93361afd1.jar:?]
	at knot/kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:48) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.2-dc3d13b93361afd1.jar:?]
	at knot/kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[org_jetbrains_kotlinx_kotlinx-co-jvm-1.10.2-dc3d13b93361afd1.jar:?]
	at knot/net.ccbluex.liquidbounce.integration.theme.ThemeManager.init(ThemeManager.kt:87) ~[JMcomicFix_v1.9.2_rc50.jar:?]
	at knot/net.ccbluex.liquidbounce.LiquidBounce.prepareGuiStage(LiquidBounce.kt:331) ~[JMcomicFix_v1.9.2_rc50.jar:?]
	at knot/net.ccbluex.liquidbounce.LiquidBounce.initializeClient(LiquidBounce.kt:172) ~[JMcomicFix_v1.9.2_rc50.jar:?]
	at knot/net.ccbluex.liquidbounce.LiquidBounce.access$initializeClient(LiquidBounce.kt:98) ~[JMcomicFix_v1.9.2_rc50.jar:?]
	at knot/net.ccbluex.liquidbounce.LiquidBounce$ClientInitializer.method_14491(LiquidBounce.kt:454) ~[JMcomicFix_v1.9.2_rc50.jar:?]
	at knot/net.minecraft.class_4013.method_29490(class_4013.java:16) ~[client-intermediary.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at knot/net.minecraft.class_4014.method_18365(class_4014.java:68) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18859(class_1255.java:164) [client-intermediary.jar:?]
	at knot/net.minecraft.class_4093.method_18859(class_4093.java:23) [client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_16075(class_1255.java:138) [client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_5383(class_1255.java:123) [client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1523(class_310.java:1296) [client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1514(class_310.java:922) [client-intermediary.jar:?]
	at knot/net.minecraft.client.main.Main.main(Main.java:267) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506) [fabric-loader-0.17.2.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72) [fabric-loader-0.17.2.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.17.2.jar:?]
Caused by: java.net.ConnectException: Failed to connect to /127.0.0.1:10721
	at knot/okhttp3.internal.connection.ConnectPlan.connectSocket(ConnectPlan.kt:282) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.ConnectPlan.connectTcp(ConnectPlan.kt:140) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.FastFallbackExchangeFinder$launchTcpConnect$1.runOnce(FastFallbackExchangeFinder.kt:141) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.concurrent.TaskRunner$runnable$1.run(TaskRunner.kt:81) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.net.ConnectException: Connection refused: getsockopt
	at java.base/sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682) ~[?:?]
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592) ~[?:?]
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.base/java.net.Socket.connect(Socket.java:751) ~[?:?]
	at knot/okhttp3.internal.platform.Platform.connectSocket(Platform.kt:147) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.ConnectPlan.connectSocket(ConnectPlan.kt:280) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.ConnectPlan.connectTcp(ConnectPlan.kt:140) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.connection.FastFallbackExchangeFinder$launchTcpConnect$1.runOnce(FastFallbackExchangeFinder.kt:141) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at knot/okhttp3.internal.concurrent.TaskRunner$runnable$1.run(TaskRunner.kt:81) ~[com_squareup_okhttp3_okhttp-jvm-5.1.0-30b975ffdb3c3de8.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

### Screenshots

[错误报告-11-9-2025_下午_9.12.19.zip](https://github.com/user-attachments/files/22275952/-11-9-2025_._9.12.19.zip)

<img width="496" height="611" alt="Image" src="https://github.com/user-attachments/assets/079ea258-45fa-4ef9-8a0d-f67be32c58d4" />

--- a/src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/ClientInteropServer.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/ClientInteropServer.kt
@@ -46,7 +46,7 @@ object ClientInteropServer {
 
     private const val DEFAULT_PORT = 15000
 
-    val port = try {
+    private var port = try {
         Socket("127.0.0.1", DEFAULT_PORT).use {
             logger.info("Default port unavailable. Falling back to random port.")
             (15001..17000).random()
@@ -56,7 +56,8 @@ object ClientInteropServer {
 
         DEFAULT_PORT
     }
-    val url = "http://127.0.0.1:$port"
+
+    val url get() = "http://127.0.0.1:$port"
 
     fun start() {
         runCatching {
@@ -82,17 +83,16 @@ object ClientInteropServer {
         }
 
         // Start the HTTP server
-        startServer()
+        this.port = startServer(this.port)
     }
 
     private var attempt = 0
-    private fun startServer(port: Int = this.port) {
-        try {
+    private fun startServer(port: Int): Int {
+        return try {
             httpServer.start(port)
         } catch (bindException: BindException) {
             if (attempt >= 5) {
                 ErrorHandler.fatal(bindException, additionalMessage = "Bind interop server")
-                return
             }
 
             // Retry with random port

